src/main/java/ca/uhn/fhir/jpa/starter/service/HelperService.java

```diff
@@ -653,6 +728,14 @@ public void refreshSyncForCurrentMonth() {
 		}
 	}
 
+	@Scheduled(cron = "0 0 23 1 * ?")
+	public void cleanupLastSyncStatusTable() {
+		// Get the current date and time
+		Timestamp currentTimestamp = new Timestamp(System.currentTimeMillis());
+		notificationDataSource = NotificationDataSource.getInstance();
+		notificationDataSource.clearLastSyncStatusTable(new Timestamp(currentTimestamp.getTime() - (30 * millisecondsInADay)));
```

Comment:
Same computation is done above also to get timestamp of 5 days before. You can make reusable function in DateUtils and pass number of days as a parameter.